packages/wdio-utils/src/test-framework/testFnWrapper.ts

```diff
@@ -81,7 +88,9 @@ export const testFrameworkFnWrapper = async function (
     }
     const duration = Date.now() - testStart
     const afterArgs = afterFnArgs(this)
-
+    if (type === 'Hook') {
+        beforeArgs.push(hookName)
```

Comment:
I believe this should be
```suggestion
        afterArgs.push(hookName)
```